inference/core/workflows/core_steps/transformations/size_measurement/v1.py

```diff
@@ -0,0 +1,171 @@
+from typing import List, Literal, Optional, Tuple, Type, Union
+
+import cv2 as cv
+import numpy as np
+import supervision as sv
+from pydantic import ConfigDict, Field, validator
+
+from inference.core.workflows.execution_engine.entities.base import (
+    Batch,
+    OutputDefinition,
+)
+from inference.core.workflows.execution_engine.entities.types import (
+    INSTANCE_SEGMENTATION_PREDICTION_KIND,
+    LIST_OF_VALUES_KIND,
+    OBJECT_DETECTION_PREDICTION_KIND,
+    StepOutputSelector,
+)
+from inference.core.workflows.prototypes.block import (
+    BlockResult,
+    WorkflowBlock,
+    WorkflowBlockManifest,
+)
+
+OUTPUT_KEY: str = "dimensions"
+TYPE: str = "roboflow_core/size_measurement@v1"
+SHORT_DESCRIPTION = (
+    "Measure the dimensions of objects in relation to a reference object."
+)
+LONG_DESCRIPTION = """
+The `SizeMeasurementBlock` is a transformer block designed to measure the dimensions of objects
+in relation to a reference object. The reference object is detected using one model,
+and the object to be measured is detected using another model. The block outputs the dimensions of the
+objects to be measured in terms of the reference object.
+"""
+
+
+class SizeMeasurementManifest(WorkflowBlockManifest):
+    model_config = ConfigDict(
+        json_schema_extra={
+            "name": "Size Measurement",
+            "version": "v1",
+            "short_description": SHORT_DESCRIPTION,
+            "long_description": LONG_DESCRIPTION,
+            "license": "Apache-2.0",
+            "block_type": "transformation",
+        }
+    )
+    type: Literal[f"{TYPE}", "SizeMeasurement"]
```

Comment:
Please just put `roboflow_core/size_measurement@v1` here, without `TYPE` var and size measurement alias